compiler/src/iree/compiler/Dialect/Encoding/Utils/Utils.h

```diff
@@ -9,10 +9,23 @@
 
 #include "iree/compiler/Dialect/Encoding/IR/EncodingTypes.h"
 #include "mlir/IR/Attributes.h"
+#include "mlir/IR/BuiltinAttributes.h"
 #include "mlir/IR/BuiltinTypes.h"
 
 namespace mlir::iree_compiler::IREE::Encoding {
 
+constexpr char kDataTilingHint[] = "iree.opt.data-tiling";
```

Comment:
Any particular reason for the hyphen? Don't we usually use an underscore instead for these constants?
```suggestion
constexpr char kDataTilingHint[] = "iree.opt.data_tiling";
```

Reply:
Not really, I did not pay much attention to it. Yes, let's use underscore for consistency. Thanks!